aws/resources/base_resource.go

```diff
@@ -30,6 +37,38 @@ func (umpl *BaseAwsResource) Nuke(_ []string) error {
 func (umpl *BaseAwsResource) GetAndSetIdentifiers(_ context.Context, _ config.Config) ([]string, error) {
 	return nil, errors.New("not implemented: GetAndSetIdentifiers")
 }
-func (umpl *BaseAwsResource) IsNukable(_ string) (bool, error) {
-	return false, errors.New("not implemented yet.")
+
+func (umpl *BaseAwsResource) GetNukableStatus(identifier string) (error, bool) {
+	val, ok := umpl.Nukables[identifier]
+	return val, ok
+}
+
+func (umpl *BaseAwsResource) SetNukableStatus(identifier string, err error) {
+	umpl.Nukables[identifier] = err
+}
+
+// nukableCheckfn : performs nukable permission verification for each ID. For each ID,
```

Comment:
I think comment should start with `VerifyNukablePermissions` to match Go conventions

Reply:
Updated.